README.md

```diff
@@ -1,2 +1,70 @@
-# react-practice-todoapp
-in this project we recreate a todo app from a tutorial
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
```

Comment:
Please add a descriptive Readme file. Plesa make sure you use the Microverse template you can find at that link: https://github.com/microverseinc/readme-template